test/customer_test.rb

```diff
@@ -3,21 +3,46 @@
 require './lib/customer'
 
 class CustomerTest < Minitest::Test
+  attr_reader :data
 
-  def test_it_has_the_expected_initialized_variables
-    skip
-    customer = Customer.new("line")
-    expected = [:id, :first_name, :last_name, :created_at, :updated_at]
-
-    expected.each do |header|
-      assert customer.respond_to?(header)
-    end
+  def setup
+    @data =   {
+                id:         "1",
```

Comment:
Unnecessary spacing detected.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.